run-tests.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+function setup() {
+    cd ./lib/validator && npm install
+}
+
+SUBCOMMAND=$1
+
+# set rust validator worker env var
+export RUST_VALIDATOR_WORKER=./target/debug/validator_worker 
+
+# RUN Tests
+if [ "$SUBCOMMAND" == "--setup" ]; then
+    echo "Setup validator tests"
+    setup    
+else
+    cargo build -p validator_worker
+    docker-compose -f docker-compose.dev.yml up
+
+    echo "Running route and integration tests"
+    ./lib/validator/test/routes.js #&& ./lib/validator/test/integration.js
```

Comment:
One more thing I noticed is that the tests depend on configuration values from JS.
Since we are running the Rust validator, they might differ, as they do now for e.g. `wait_time`, `aggr_throttle`

Reply:
those vars shouldn't matter as we are running the tests in development mode